attribute-event-requests/INDI-event.md

```diff
@@ -14,12 +14,13 @@ Guides for using this document can be found in the associated [README.md](README
 | `BASM` | 4.0 | [Bat Mitzvah](#bat-mitzvah) | |
 | `BIRT` | 3.0 | [Birth](#birth) | |
 | `BLES` | 3.0 | Blessing | |
-| `BURI` | 3.0 | [Burial](#burial) | |
+| `BURI` | 3.0 | [Burial](#burial) | one of several types of `BURI` |
 | `CENS` | 3.0 | Census | |
 | `CHR`  | 3.0 | [Christening](#christening) | |
 | `CONF` | 3.0 | Confirmation | |
-| `CREM` | 5.4 | [Cremation](#cremation) | |
+| `CREM` | 5.4 | [Cremation](#cremation) | sometimes encoded as `BURI` prior to 5.4 |
 | `DEAT` | 3.0 | [Death](#death) | |
+| `BURI` | 3.0 | [Depositing remains](#depositing-remains) | |
```

Comment:
I do find it confusing to see BURI in both line 17 and line 23, with two different definitions, both saying since 3.0.  Perhaps line 17 shouldn't be BURI, since there was no tag specifically for burial, as opposed to depositing remains in general?